README.md

```diff
@@ -41,40 +43,43 @@ Minishift requires a hypervisor to run the virtual machine containing OpenShift.
 host OS, you have the choice of the following hypervisors:
 
 * OS X
-    * [xhyve](https://github.com/mist64/xhyve), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion)
+    * [xhyve](https://github.com/mist64/xhyve) (default), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion)
 * Linux
-    * [KVM](./docs/docker-machine-drivers.md#kvm-driver) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+    * [KVM](./docs/docker-machine-drivers.md#kvm-driver) (default) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 * Windows
-    * [Hyper-V](https://technet.microsoft.com/en-us/library/mt169373.aspx) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+    * [Hyper-V](https://technet.microsoft.com/en-us/library/mt169373.aspx) (default) or [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
-Minishift ships with drivers for VirtualBox and VMware Fusion out of the box. Other drivers
-require manual installation, see [docker machine drivers installation](./docs/docker-machine-drivers.md) for more details.
+Make sure that the hypervisor is installed and enabled on your system before you install Minishift.
 
-Drivers can be selected via the `--vm-driver=xxx` flag of `minishift start` as mentioned in the [Starting Minishift](#starting-minishift) section below. See the [Troubleshooting](/docs/troubleshooting.md#kvm-drivers) guide, in case you encounter any issues.
+**Important:**
 
-**Note:**
-- For most hypervisors, VT-x/AMD-v virtualization must be enabled in the BIOS. For Hyper-V, however,
-it needs to be disabled.
-- We recommend that you use `Virtualbox >= 5.1.12` on Windows to avoid the issue
+- On some operating systems, KVM and xhyve require specific installation and configuration steps.
+For more information, see [docker machine drivers installation](./docs/docker-machine-drivers.md).
+- For most hypervisors, you must enable VT-x/AMD-v virtualization in the BIOS. However, if you want
```

Comment:
I had to enable virtualization in my BIOS before I could run minishift, but honestly I don't remember which type it was. I just tabbed thru all the options until I saw "virtualization"